### DISCUSSION TITLE
Bulk importing from Roam Research (or similar)?

### DISCUSSION THREAD

**Discussion by seanvosler** [knowledgefutures/pubpub#2060] [Support]
Title basically - I don't see any documentation on importing bulk content, thoughts? Feelings? Ideas?

**Comment by gabestein** [comment]
Hi Sean — thanks for getting in touch, and sorry for the delay getting back to you.

We have a bulk import tool we can run behind the scenes that's pretty flexible, though I don't think we've done anything from Roam yet. Is it possible to convert Roam pages to markdown docs? That may be the easiest path forward.
